docker/mini/Dockerfile

```diff
@@ -40,9 +37,6 @@ COPY docker/mini/vttablet-mini-up.sh /vt/dist/scripts/vttablet-mini-up.sh
 RUN echo "hostname=127.0.0.1" >> /vt/dist/scripts/env.sh
 RUN cat /vt/dist/scripts/env.sh | egrep "^alias" >> /etc/bash.bashrc
 
-COPY --from=base /vt/bin/vtctl /vt/bin/
-COPY --from=base /vt/bin/mysqlctl /vt/bin/
```

Comment:
Those two binaries are already present in `vitess/lite` no need to copy them and no need for a base or bootstrap image.